### PR TITLE
Fixes #23566 - Count and show icons for suse errata

### PR DESCRIPTION
**Pull request by mdellweg**

### CODE REVIEW

**Comment by sbernhard** [comment]
[test katello]

File: app/models/katello/erratum.rb [hunk 1]
```diff
@@ -3,8 +3,8 @@ class Erratum < Katello::Model
     include Concerns::PulpDatabaseUnit
 
     SECURITY = "security".freeze
-    BUGZILLA = "bugfix".freeze
-    ENHANCEMENT = "enhancement".freeze
+    BUGZILLA = ["bugfix", "recommended"].freeze
+    ENHANCEMENT = ["enhancement", "optional"].freeze
```
Comment (jlsherrill) [inline app/models/katello/erratum.rb]: For simplicity, maybe we should put all the types in an array?

Comment (mdellweg) [reply app/models/katello/erratum.rb]: sure

File: app/models/katello/erratum.rb [hunk 2]
```diff
@@ -3,8 +3,8 @@ class Erratum < Katello::Model
...
 
     TYPES = [SECURITY, BUGZILLA, ENHANCEMENT].freeze
```
[the remark this answers is not in the document]
Comment (mdellweg) [reply app/models/katello/erratum.rb]: not at all

[the remark this answers is not in the document]
Comment (mdellweg) [reply app/models/katello/erratum.rb]: Is it used for anything?

**Comment by jlsherrill** [comment]
Thanks @mdellweg !

**Pull request merged**